### PR TITLE
[devtools-translate]: Chinese - What's new in DevTools (Chrome 95)

### PR DESCRIPTION
**Pull request by hanselfmu**
This PR adds translations for "What's New in DevTools 95" from English to Simplified Chinese.

### CODE REVIEW

File: site/zh/blog/new-in-devtools-95/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ authors:
 date: 2021-09-20
 updated: 2021-09-20
 description:
-  "New CSS length authoring tools, hide issues in the Issues tab, improved the display of properties, and more."
+  "新的 CSS 长度编辑工具、在“问题”标签页隐藏问题、提升属性展示以及更多。"
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-95/index.md:9]: **提升属性展示** seems to be strange. It's very common that some people use **提升** with some words such as **提升作用域**. So I think **改善** or **优化** is a better choice than **提升**.

File: site/zh/blog/new-in-devtools-95/index.md [hunk 2]
```diff
@@ -29,53 +29,68 @@ tags:
 
 
 <!-- ## New CSS length authoring tools {: #length } -->
+## 新的 CSS 长度编辑工具 {: #length }
 
 <!-- DevTools added an easier yet flexible way to update lengths in CSS! -->
+DevTools 添加了一种更简单且灵活的方式来更新 CSS 长度！ 
 
 <!-- In the **Styles** pane, look for any CSS property with length (e.g. `height`, `padding`). -->
+在“样式”边栏中，找到任意一个含有长度的 CSS 属性。
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-95/index.md:38]: > In the **Styles** pane, look for any CSS property with length (e.g. `height`, `padding`)

The current translation of the above sentence is still in incomplete. Namely, you have ignored some content such as **(e.g. `height`, `padding`)**.

File: site/zh/blog/new-in-devtools-95/index.md [hunk 3]
```diff
@@ -29,53 +29,68 @@ tags:
...
 
 <!-- Hover over the unit type, and notice the unit type is underlined. Click on it to select a unit type from the dropdown. -->
+将鼠标悬停在长度单位上，就会发现它被下划线标明。点击它即可在弹出的下拉列表中选择新的长度单位。
 
 {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/vWiU9o1DxsOpWXM0SrBa.mp4", autoplay="true", muted="true", loop="true", class="screenshot" %}
 
 <!-- Hover over the unit value, and your mouse pointer is changed to horizontal cursor. Drag horizontally to increase or decrease the value. To adjust the value by 10, hold the <kbd>Shift</kbd> key when dragging. -->
+将鼠标悬停在长度数值上，就会发现您的鼠标指针变成了横向移动的光标。横向拖拽光标即可加减长度数值。如果想以10来调整长度数值，可以在拖拽的同时按下 <kbd>Shift</kbd> 键。
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-95/index.md:46]: > 如果想以10来调整长度数值

The number 10 inside the above sentence needs both spaces.

File: site/zh/blog/new-in-devtools-95/index.md [hunk 4]
```diff
@@ -88,44 +103,52 @@ link.search = `?access=${link.access}`;
 ```
 
 <!-- Try logging `link` in the **Console**. Own properties are now bold and sorted first. These changes make it easier to spot custom properties, especially for [Web APIs](https://developer.mozilla.org/en-US/docs/Web/API) (e.g. `URL`) with many inherited properties. -->
+试着在“控制台”中打出 `link`。自由属性现在是加粗并优先排序的。这些改动会让自定义属性更容易发现，尤其对于有着很多继承属性的 [Web APIs](https://developer.mozilla.org/en-US/docs/Web/API)（例如 `URL`）。
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-95/index.md:106]: > 自由属性现在是加粗并优先排序的

It seems to be a typo. So you should replace **自由** with **自有** to fix typo.

Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-95/index.md:106]: > 自由属性现在是加粗并优先排序的

It seems to be a typo. So you should replace **自由** with **自有** to fix the typo above.

Comment (hanselfmu) [reply site/zh/blog/new-in-devtools-95/index.md:106]: Yes, thanks!

**Comment by liuliangsir** [comment]
@hanselfmu I'll review again soon.

File: site/zh/blog/new-in-devtools-95/index.md [hunk 5]
```diff
@@ -88,44 +103,52 @@ link.search = `?access=${link.access}`;
 ```
 
 <!-- Try logging `link` in the **Console**. Own properties are now bold and sorted first. These changes make it easier to spot custom properties, especially for [Web APIs](https://developer.mozilla.org/en-US/docs/Web/API) (e.g. `URL`) with many inherited properties. -->
+试着在“控制台”中打出 `link`。自有属性现在是加粗并优先排序的。这些改动会让自定义属性更容易发现，尤其对于有着很多继承属性的 [Web APIs](https://developer.mozilla.org/en-US/docs/Web/API)（例如 `URL`）。
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-95/index.md:106]: > 这些改动会让自定义属性更容易发现

I think the above sentence can be inserted a word(**被**) followed by **发现**.

File: site/zh/blog/new-in-devtools-95/index.md [hunk 6]
```diff
@@ -29,53 +29,68 @@ tags:
...
 <!-- ## Improved the display of properties {: #properties } -->
+## 改善属性展示 {: #properties }
 
 <!-- DevTools improve the display of properties by: -->
+DevTools 通过以下方式改善了属性的展示：
 
 <!-- - Always bold and sort own properties first in the **Console**, **Sources** panel and **Properties** pane. 
 - Flatten the properties display in the **Properties** pane. -->
+- 在控制台、“来源”面板和“属性”边栏中，永远加粗并优先排序自有属性。
+- 在“属性”边栏中扁平化属性展示。
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-95/index.md:90]: > 在“属性”边栏中扁平化属性展示

The sentence above can be represented by **在“属性”边栏中扁平化展示属性**

File: site/zh/blog/new-in-devtools-95/index.md [hunk 7]
```diff
@@ -88,44 +103,52 @@ link.search = `?access=${link.access}`;
...
 
-Chromium issues: [1076820](https://crbug.com/1076820), [1119900](https://crbug.com/1119900)
+Chromium 议题: [1076820](https://crbug.com/1076820), [1119900](https://crbug.com/1119900)
 
 
 <!-- ## Lighthouse 8.4 in the Lighthouse panel {: #lighthouse } -->
+## Lighthouse 面板中的 Lighthouse 8.4 {: #lighthouse }
 
 <!-- The **Lighthouse** panel is now running Lighthouse 8.4. Lighthouse will now detect if the [Largest Containful Paint (LCP)](https://web.dev/lcp) element was a lazy-loaded image and recommend removing the `loading` attribute from it. -->
+**Lighthouse** 面板现已运行 Lighthouse 8.4。Lighthouse 现在可以检测到某 [Largest Containful Paint (LCP)](https://web.dev/lcp) 元素是一个延迟加载的图片，并建议移除该图片的 `loading` 属性。
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-95/index.md:126]: >现在可以检测到某 [Largest Containful Paint (LCP)](https://web.dev/lcp) 元素是一个延迟加载的图片

I think the above sentence can be inserted two words(**是否**) followed by **是一个延迟加载的图片**.

File: site/zh/blog/new-in-devtools-95/index.md [hunk 8]
```diff
@@ -143,20 +166,24 @@ Chromium issues: [1246245](https://crbug.com/1246245), [1245481](https://crbug.c
 
 
 <!-- ## Improved UI for DevTools command menu {: #command-menu } -->
+## DevTools 命令菜单的用户交互优化 {: #command-menu }
 
 <!-- Did you find it hard to search for a file in the [Command Menu](/docs/devtools/command-menu/#open)? Good news for you, the **Command Menu** user interface is now enhanced!  -->
+您是否发现在[命令菜单](/docs/devtools/command-menu/#open)中搜索一个文件很困难呢？好消息来了，**命令菜单**的用户交互现在得到了改善！
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-95/index.md:172]: > 您是否发现在[命令菜单](/docs/devtools/command-menu/#open)中搜索一个文件很困难呢？

The sentence above can be better represented by **您是否发现要想在[命令菜单](/docs/devtools/command-menu/#open)中搜索一个文件，这本身就是一件很困难的事情**

**Review by Z-byte-prog — APPROVED** [review]

File: site/zh/blog/new-in-devtools-95/index.md [hunk 9]
```diff
@@ -29,53 +29,68 @@ tags:
...
+## 新的 CSS 长度编辑工具 {: #length }
 
 <!-- DevTools added an easier yet flexible way to update lengths in CSS! -->
+DevTools 添加了一种更简单且灵活的方式来更新 CSS 长度！ 
 
 <!-- In the **Styles** pane, look for any CSS property with length (e.g. `height`, `padding`). -->
+在“样式”边栏中，找到任意一个含有长度的 CSS 属性（例如 `height`、`padding`）。
 
 <!-- Hover over the unit type, and notice the unit type is underlined. Click on it to select a unit type from the dropdown. -->
+将鼠标悬停在长度单位上，就会发现它被下划线标明。点击它即可在弹出的下拉列表中选择新的长度单位。
```
[the remark this answers is not in the document]
Comment (hanselfmu) [reply site/zh/blog/new-in-devtools-95/index.md]: I've updated this to:
“将鼠标悬停在长度单位上，会出现一条下划线来标明它”

What do you think?

[the remark this answers is not in the document]
Comment (liuliangsir) [reply site/zh/blog/new-in-devtools-95/index.md]: It looks good to me

**Pull request merged**